### ISSUE TITLE
Fully specify dependencies

### ISSUE THREAD

**Issue by h-vetinari** [containers/python-podman#36]
I realise a lot is in flux here with and after #25, but just mentioning that I tried to install from git, and the installation process does not trigger any dependencies to be pulled in (I'm sure that once published to PyPI this would work already a bit better), and the metadata seems to be missing some packages.

First error I got was missing package `varlink`, and while installing that, I got:
```
>pip install varlink
Collecting varlink
  Downloading https://files.pythonhosted.org/packages/80/4f/8ac1e3fd7f766b68a5273926faf810d7a2d81863ce0a57c74b47461dd353/varlink-30.1.2-py2.py3-none-any.whl
ERROR: podman 0.0.1 requires psutil, which is not installed.
ERROR: podman 0.0.1 requires python-dateutil, which is not installed
```

So it seems `psutil` / `python-dateutil` are specified, but not varlink, and neither the missing package error I got after installing all that.
```
>>> import podman
Traceback (most recent call last):
[...]
ModuleNotFoundError: No module named 'fcntl'
[...]
>pip install fcntl
Collecting fcntl
  ERROR: Could not find a version that satisfies the requirement fcntl (from versions: none)
ERROR: No matching distribution found for fcntl
```

Not sure what else might be missing, since I cannot install fcntl from either pip or conda.

**Comment by 4383** [comment]
Hello

`python-podman` package is built by using `pbr` who will  [automatically use the `requirements.txt` file](https://docs.openstack.org/pbr/latest/user/using.html#requirements).

This file [already contains varlink](https://github.com/containers/python-podman/blob/master/requirements.txt) etc... so normally when you install `python-podman` from pypi it automatically install varlink too 

Concerning [fcntl this module is a part of the stdlib](https://docs.python.org/3.5/library/fcntl.html) so you don't need to install it and you can't find it on pypi, it's already available with your python installation.

Normally an installation from git work fine too with pbr and in a similar way than a pypi install.

First can you try this before your install:
```shell
$ # from your local clone repository
$ python -m pip install --user pbr
$ python setup.py install
$ # normally dependencies will be installed too...
```

Let's me know if it work now.

Also which python version are in use here?

Can you give the commands that you use?

Thanks :)

**Comment by h-vetinari** [comment]
@4383 
I used the install instructions from the readme:
```
cd ~/python-podman
python3 setup.py clean -a && python3 setup.py sdist bdist
python3 setup.py install --user
```

And for me, `fnctl` is not part of the stdlib for Python 3.7.

**Comment by h-vetinari** [comment]
OK, on a linux machine, with varlink,, pbr, psutil & python-dateutil installed, `python setup.py install` works.

**Comment by 4383** [comment]
@h-vetinari The `README.md` documentation to install podman work well, my previous comment seems to fix your issue so if it's ok for you can we close this issue?
